src/components/Animator/components/AnimatorParent.js

```diff
@@ -15,13 +15,13 @@ class AnimatorParent extends React.Component {
   }
 
   render() {
-    const {className, dataHook} = this.props;
+    const {className, dataHook, skipMountTransition} = this.props;
     const animatorProps = formatProps(this.props);
     animatorProps.debug = this.state.debug;
     return (
       <TransitionGroup data-hook={dataHook} className={className}>
         {animatorProps.children.map((item, index) =>
-          <CSSTransitionWrapper key={item.key || index} index={index} {...item.props} animatorProps={animatorProps}>
+          <CSSTransitionWrapper skipMountTransition={skipMountTransition} key={item.key || index} index={index} {...item.props} animatorProps={animatorProps}>
```

Comment:
I don't see in the API documentation any reference for the new prop added. Also, I think that adding an example can be useful for this as well.

Reply:
Added:
https://wix-wix-animations-pr-17.surge.sh/?selectedKind=Animator&selectedStory=Full%20API&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel